single-line-diagram-core/src/main/java/com/powsybl/sld/svg/FlowArrow.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.sld.svg;
+
+import java.util.Optional;
+
+import com.powsybl.sld.svg.DiagramLabelProvider.Direction;
+
+/**
+ * @author Giovanni Ferrari <giovanni.ferrari at techrain.eu>
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class FlowArrow {
+
+    private final Direction direction;
+    private final String leftLabel;
+    private final String rightText;
```

Comment:
I think we can use this PR to add a componentType String, to specify which kind of arrow it is. It will then be used to get the corresponding component (and style), removing the iArrow clumsy parameter. This allows to have arrows drawn completely differently based on the type. For the basic case of 2 arrows, one for P, one for Q, we then need to have 2 arrow components in the components.json, one with sld-arrow-p style, the other with sld-arrow-q.